demisto_sdk/commands/unify/generic_module_unifier.py

```diff
@@ -0,0 +1,61 @@
+"""
+Unifies a GenericModule object with it's Dashboards
+"""
+import json
+import os
+from typing import Dict, Optional
+
+
+def find_dashboard_by_id(pack_path: str, dashboard_id: str) -> Optional[Dict]:
+    """
+    Search for dashboard with the given id in the given pack path.
+
+    Args:
+        pack_path: a pack to search in
+        dashboard_id: dashboard id to search for
+
+    Returns: if found - the content of the dashboard, else - None.
+    """
+    for file_name in os.listdir(pack_path):
+        file_path = os.path.join(pack_path, file_name)  # TODO: need to make sure how exactly the dashboards located in the pack
+        if file_path.endswith('.json') and file_name.startswith('dashboard'): # it's a dashboard
+            with open(file_path) as f:
+                dashboard = json.load(f)
```

Comment:
`json.load` should do the trick here.

Reply:
json.load gets file objects as arguments, and json.loads gets a json string as an argument and not the path. 
I think I'll leave it as it is.